src/runtime-rs/crates/resource/src/cpu_mem/initial_size.rs

```diff
@@ -125,7 +148,7 @@ impl InitialSizeManager {
             // plus the default_memory here, instead of overriding it.
             // (if we override the default_memory here, and user apllications still
             // use memory as they orignally expected, it would be easy to OOM.)
-            hv.memory_info.default_memory += self.resource.mem_mb;
+            hv.memory_info.default_memory = self.resource.mem_mb;
```

Comment:
Do we want to override the config file anytime the `sandbox-memory` annotation is present?

I can see how it could make sense in the context of Sandbox API, just making sure this is a conscious decision.

Also, if so, could you update the reasoning in the comment above the changed line as well?

Reply:
@pmores   Good catch, actually it's was changed by accident. I had revert this change.